nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/request/FlowMetricsReportingStrategy.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.web.api.request;
+
+
+import org.apache.nifi.components.AllowableValue;
+
+import static org.apache.nifi.prometheusutil.PrometheusMetricsUtil.METRICS_STRATEGY_COMPONENTS;
+import static org.apache.nifi.prometheusutil.PrometheusMetricsUtil.METRICS_STRATEGY_PG;
+
+public enum FlowMetricsReportingStrategy {
+    ONLY_PROCESS_GROUPS(METRICS_STRATEGY_PG),
```

Comment:
The display name for `METRICS_STRATEGY_PG` is `All Process Groups`, so it would be better to align this `enum` name:

```suggestion
    ALL_PROCESS_GROUPS(METRICS_STRATEGY_PG),
```